easylistportuguese/easylistportuguese_specific_block.txt

```diff
@@ -97,4 +97,5 @@
 ||ecoregional.com.br/wp-content/uploads/*/jardim-pq-*.jpg
 ||ecoregional.com.br/wp-content/plugins/icegram/
 ||notebookcheck.info/fileadmin/Sonstiges/ama_abn2_$xmlhttprequest
-||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
\ No newline at end of file
+||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
+||s3.amazonaws.com/strapi-uploads/*.png$image,domain=maisesports.com.br
```

Comment:
Hey @felippesantos46 , does this filter works for you? It doesn't for me. 

These images you want to block are served like this:
`https://maisesports.com.br/_next/image?url=https%3A%2F%2Felasticbeanstalk-us-east-1-909474674380.s3.amazonaws.com%2Fstrapi-uploads%2F%2FBG_ad_03_44a51ac7b8.png&w=828&q=75`

So, the url with the image is actually a parameter passed to maisesports.com.br.  I would rather suggest something like this: 

`/_next/image?url=*s3.amazonaws.com` and if wanted maybe make it specific for the domain.

Reply:
Hey, my bad, i forgot to change to yours.
On this matter just this one is necessary: `/_next/image?url=*s3.amazonaws.com`
As they had a bug back then in their webpage and the following was fixed, we do not need this one: `maisesports.com.br##a[href^="https://mais.gg/"]`

If i need to change something else, just let me know !